tests/test_sessions_api.py

```diff
@@ -97,3 +99,79 @@ def test_manual_session_aggregation(self):
         assert data["data"][0]["sessions_errored"] == 10
         assert data["data"][0]["users"] == 2
         assert data["data"][0]["users_errored"] == 1
+
+    def generate_session_events(self):
+        processor = self.storage.get_table_writer().get_stream_loader().get_processor()
+        meta = KafkaMessageMetadata(
+            offset=1, partition=2, timestamp=datetime(1970, 1, 1)
+        )
+        template = {
+            "session_id": "00000000-0000-0000-0000-000000000000",
+            "distinct_id": "b3ef3211-58a4-4b36-a9a1-5a55df0d9aaf",
+            "duration": None,
+            "environment": "production",
+            "org_id": 1,
+            "project_id": 2,
+            "release": "sentry-test@1.0.0",
+            "retention_days": settings.DEFAULT_RETENTION_DAYS,
+            "seq": 0,
+            "errors": 0,
+            "received": datetime.utcnow().timestamp(),
+            "started": self.started.timestamp(),
+        }
+        events = [
+            processor.process_message(
+                {
+                    **template,
+                    "status": "exited",
+                    "duration": 1947.49,
+                    "session_id": "8333339f-5675-4f89-a9a0-1c935255ab58",
+                },
+                meta,
+            ),
+            processor.process_message(
+                {**template, "status": "exited", "quantity": 5}, meta,
+            ),
+            processor.process_message(
+                {**template, "status": "errored", "errors": 1, "quantity": 2}, meta,
```

Comment:
Why would you set the errors number on a pre aggregated session when the number of errors is calculated by summing the quantity of sessions that have status == errored ?

Reply:
Like you mentioned, there is no real need to do so, I just do it out of convention, to keep things consistent really.